bpf/lib/policy.h

```diff
@@ -30,7 +30,6 @@
  * - IdentityUnknown		(0)
  * - ReservedIdentityHost	(1)
  * - ReservedIdentityWorld	(2)
- * - ReservedIdentityCluster	(3)
```

Comment:
It might be better to leave this marked in some way? I would have to re-use this identity and forget that it meant something to older cilium versions (same for the actual define in node_config.h)